simulation/src/country_data.proto

```diff
@@ -17,5 +17,6 @@ message DailyStats {
 message CountryData {
   string name = 1;
   string short_name = 3;
+  uint32 population = 4;
```

Comment:
I suggest uint64 if we would like to add whole world.

Reply:
Done